interoptest/src/nodejsservice/server.js

```diff
@@ -38,10 +38,10 @@ function main () {
   //enableGrpcPlugin(tracer);
 
   // Start GRPC Server
-  grpcServer.start(interop.ServicePort.NODEJS_GRPC_BINARY_PROPAGATION_PORT);
+  grpcServer.start(interop.ServicePort.NODEJS_GRPC_BINARY_PROPAGATION_PORT, '0.0.0.0');
 
   // Start HTTP Server
-  httpServer.start(interop.ServicePort.NODEJS_HTTP_TRACECONTEXT_PROPAGATION_PORT);
+  httpServer.start(interop.ServicePort.NODEJS_HTTP_TRACECONTEXT_PROPAGATION_PORT, '0.0.0.0');
```

Comment:
Optional nit : it make sense to alter DEFAULT_HOST (util.js), instead of passing new default host here.

Reply:
DEFAULT_HOST is also used in for sending request. see [here](https://github.com/census-ecosystem/opencensus-experiments/blob/5dd202bb98f8fa22b2376d40668275dca29525cd/interoptest/src/nodejsservice/src/testservice/service-hopper.js#L37)

If the host is not present then it should send to localhost and not '0.0.0.0'.